src/classes/RD_RecurringDonations.cls

```diff
@@ -728,9 +699,10 @@ public class RD_RecurringDonations {
         //opps for updating
         list<Opportunity> updateOppsList = new list<Opportunity>();
         
+        // TODO: refactor this loop out into its own method so i can use it elsewhere as well
```

Comment:
still planning on this TODO?  we should avoid checking in any TODO's into master.

Reply:
removed. no longer planned. thanks for the catch.